extensions/binding/org.eclipse.smarthome.binding.astro.test/src/test/groovy/org/eclipse/smarthome/binding/astro/handler/test/AstroReadOnlyTest.groovy

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
The usage of groovy is deprecated in this project (#3305). Please don't add any new ones! 

By the way - how is this related to ZonedDateTime? The only this this does is with lots of efforts asserting that _no_ state updated was issued. So from my perspective you can drop it completely. It tests nothing and only slows down the build (a little).